src/main/java/edu/cornell/kfs/vnd/businessobject/VendorEmployeeComparisonResult.java

```diff
@@ -0,0 +1,83 @@
+package edu.cornell.kfs.vnd.businessobject;
+
+import java.time.LocalDate;
+
+import org.kuali.kfs.krad.bo.TransientBusinessObjectBase;
+
+public class VendorEmployeeComparisonResult extends TransientBusinessObjectBase {
+
+    private static final long serialVersionUID = 1L;
+
+    private String vendorId;
+    private String employeeId;
+    private String netId;
+    private Boolean active;
+    private LocalDate hireDate;
+    private LocalDate terminationDate;
+    private LocalDate terminationDateGreaterThanProcessingDate;
+
+    public String getVendorId() {
+        return vendorId;
+    }
+
+    public void setVendorId(String vendorId) {
+        this.vendorId = vendorId;
+    }
+
+    public String getEmployeeId() {
+        return employeeId;
+    }
+
+    public void setEmployeeId(String employeeId) {
+        this.employeeId = employeeId;
+    }
+
+    public String getNetId() {
+        return netId;
+    }
+
+    public void setNetId(String netId) {
+        this.netId = netId;
+    }
+
+    public boolean isActive() {
+        return active != null && active.booleanValue();
+    }
+
+    public boolean isInactive() {
+        return active != null && !active.booleanValue();
```

Comment:
These result objects will ultimately be generated from a .csv file returned by Workday, so I had set them up as `Boolean` properties to help clarify whether a null value was sent back to us. However, since we're expecting Workday to return non-null values in this field, I'll look into updating this to use primitive booleans instead, and then have the conversion either translate nulls to false or treat nulls as errors.

Reply:
Actually, upon further review, I remembered that I had set up the reporting so that it would warn about missing data in this field, so it needs to remain `Boolean` to simplify that reporting.